Multisig/UI/ClaimToken/ClaimingAmountViewController/ClaimTokensViewController.swift

```diff
@@ -0,0 +1,496 @@
+        completion()
+    }
+
+    var isMax: Bool {
+        guard
+            let row = rows.firstIndex(of: .claimingAmount),
+            let cell = tableView.cellForRow(at: IndexPath(row: row, section: 0)) as? ClaimedAmountInputCell
+        else {
+            return false
+        }
+        return cell.isMax
+    }
+
+    var inputAmount: Sol.UInt128? {
+        guard
+            let row = rows.firstIndex(of: .claimingAmount),
+            let cell = tableView.cellForRow(at: IndexPath(row: row, section: 0)) as? ClaimedAmountInputCell
+        else {
+            return nil
+        }
+        return cell.value
+    }
+
+    // edit selected delegate
+    @IBAction private func editButtonTouched(_ sender: Any) {
+        onEditDelegate()
+    }
+
+    // pull-to-refresh, initial reload
+    override func reloadData() {
+        super.reloadData()
+
+        timestamp = Date().timeIntervalSince1970
+
+        keyboardBehavior.hideKeyboard()
+
+        controller.asyncFetchData(account: safe.addressValue) { [weak self] result in
+            guard let self = self else { return }
+            do {
+                self.claimData = try result.get()
+                self.onSuccess()
+            } catch {
+                self.onError(GSError.error(description: "Failed to load data", error: error))
+            }
+        }
+    }
+
+}
+
+extension ClaimTokensViewController: UITableViewDelegate, UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        rows.count
+    }
+
+    func formatted(amount: Sol.UInt128) -> String {
+        let decimal = BigDecimal(Int256(amount.big()), 18)
+        let value = tokenFormatter.string(from: decimal)
+        let amount = value + " SAFE"
+        return amount
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let row = rows[indexPath.row]
+
+        switch row {
+        case .claimableNow:
+            let cell = tableView.dequeueCell(AllocationBoxCell.self)
+            cell.headerText = "Claim now"
+            cell.titleText = "Total"
+            cell.tooltipHostView = view
+
+            guard let claimData = claimData else {
+                // defaults when data not loaded
+                cell.valueText = "..."
+                cell.titleTooltipText = nil
+                cell.headerTooltipText = nil
+
+                // must be set at the end to update values
+                cell.style = .darkUser
+                return cell
+            }
+
+
+            let userAllocation = claimData.allocationsData.first {
+                $0.allocation.tag.contains("user")
+            }
+            let ecosystemAllocation = claimData.allocationsData.first {
+                $0.allocation.tag.contains("ecosystem")
+            }
+
+            if let userAllocation = userAllocation, let ecosystemAllocation = ecosystemAllocation, claimData.allocationsData.count == 2 {
```

Comment:
For future refactor pr:
- Let's make it to account for all user allocations, so that it would work for future. 
- Maybe it make sense to refactor with getting the data first and then creating other cells